webhook/certificates/certificates.go

```diff
@@ -67,8 +75,19 @@ func (r *reconciler) reconcileCertificate(ctx context.Context) error {
 	} else if _, haskey := secret.Data[certresources.CACert]; !haskey {
 		logger.Infof("Certificate secret %q is missing key %q", r.secretName, certresources.CACert)
 	} else {
-		// It has all of the keys, it's good.
-		return nil
+		// Check the expiration date of the certificate to see if it needs to be updated
+		cert, err := tls.X509KeyPair(secret.Data[certresources.ServerCert], secret.Data[certresources.ServerKey])
+		if err != nil {
+			logger.Errorf("error creating pem from certificate and key: %v", err)
```

Comment:
```suggestion
			logger.Errorf("Error creating PEM from certificate and key: %v", err)
```
Logger errors are complete sentences.
Since we're ignoring them, should they rather be `Warnf`? Same below?

Reply:
Will do, wasn't sure which we should use. Both were suggested so I went with the same error type that was being used further up. I'll change over to Warnf.

Reply:
Error will print stacktrace etc. 
Given you ignore the error, I'd go with Infof/Warnf. I have no preference which.